buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijVersionHelper.kt

```diff
@@ -15,7 +14,7 @@ object IntellijVersionHelper {
     if (versionKey == null) {
       return emptyList()
     }
-    val regex = "/^[0-9].*$/".toRegex()
+    val regex = "^[0-9].*$".toRegex()
```

Comment:
Was it a bug?

Reply:
Yes... in Groovy `/.../` syntax is a regex, no quotes needed; nothing like that in Kotlin tho